spec/models/agents/post_agent_spec.rb

```diff
@@ -311,8 +311,21 @@
   end
 
   describe "#working?" do
+    it "checks if there was an error" do
+      log('Error!')
+      expect(AgentLog.count).to eq(1)
```

Comment:
`@checker.logs.count` could work here. If it also produces test failures you could remove this line.